opm/utility/ECLGraph.cpp

```diff
@@ -274,6 +279,19 @@ namespace {
                 std::vector<T>
                 scatterToGlobal(const std::vector<T>& x) const;
 
+                /// Map input vector to active grid cells.
+                ///
+                /// \param[in] x Input vector, defined on the explicitly
+                ///              active cells, all global cells or some
+                ///              other subset (e.g., all non-neighbouring
+                ///              connections).
+                ///
+                /// \return Input vector mapped to active cells or unchanged
```

Comment:
The caller must ultimately be aware of which use case it is invoking, so I do not really see the problem? I can live with clear explanations however.